app/controllers/admin_events_controller.rb

```diff
@@ -63,4 +62,16 @@ def destroy
     def get_event
       @event = Event.find(params[:id])
     end
+
+    def inform_applicants_country
+      User.where(country: @event.country).where(country_email_notifications: true).each do |user|
+        UserNotificationsMailer.new_local_event(@event, user).deliver_later
+      end
+    end
+
+    def inform_applicants_field_of_interest
+      @event.interested_users.where(tag_email_notifications: true).each do |user|
+        UserNotificationsMailer.new_local_event(@event, user).deliver_later
```

Comment:
instead of calling new_local_event we want to call the new mailer function new_field_specific_event here